react-basic-practice/package.json

```diff
@@ -16,7 +16,8 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-router-dom": "^6.4.5",
-    "uuid": "^9.0.0"
+    "uuid": "^9.0.0",
+    "vite-project": "link:"
```

Comment:
What is this?

Reply:
maybe I installed it by mistake when I reinstalled the support tool, I will remove it